keyboards/lily58/config.h

```diff
@@ -21,8 +21,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 #include <serial_config.h>
```

Comment:
The stuff from the serial_config.h file should be moved into this file, and that file removed.

Reply:
The OLEDs appear to be non-functional for me in this release

Reply:
I would, but [ ]  My code follows the code style of this project. 😅 

I'll see if I can refactor some of the OLED code to work better in the keymap